extensions/sns/src/commands/prepare_canisters/remove_nns_root.rs

```diff
@@ -0,0 +1,26 @@
+//! Code for executing `dfx sns prepare-canisters remove-nns-root`
+use candid::Principal;
+use clap::Parser;
+use dfx_extensions_utils::call_extension_bundled_binary;
+use std::ffi::OsString;
+use std::path::Path;
+
+/// `dfx sns prepare-canisters remove-nns-root` command line arguments.
+#[derive(Parser)]
+pub struct RemoveNnsRootOpts {
+    // TODO support multiple canisters?
```

Comment:
It is kind of funny that the command is called `prepare-canisters` but the CLI accepts exactly one canister ID.

Reply:
Ideally, we wouldn't need to have a flag that would need to be repeated, and `prepare-canisters` could take positional arguments and not flag arguments, e.g.

```
dfx sns prepare-canisters add-nns-root <CANISTER_ID>
```

instead of 

```
dfx sns prepare-canisters add-nns-root --canister-id <CANISTER_ID> <CANITER_ID>

or 

dfx sns prepare-canisters add-nns-root --canister-id <CANISTER_ID> --canister-id <CANITER_ID>
```

As I understand it (asked [here](https://github.com/dfinity/dfx-extensions/pull/38#discussion_r1287693739)), the extensions.json doesn't allow this (or I am still unaware of the correct incantation).

This is just a matter of personal taste to me TBH, as I think its confusing to explain and for users to get right, but it would be a very simple change to just add `#[arg(long, multiple_values = true)]` here. WDYT?

Reply:
Looks like theres been a development in between me starting to write this comment and publishing it :D I'll report back with what I find.

Reply:
With the help of @smallstepman's latest changes I was able to specify multiple canister ids!